ftw/solr/profiles/default/componentregistry.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0"?>
+<componentregistry>
+ <adapters/>
+ <utilities>
+  <utility
```

Comment:
this will be a persistent utility? why is this necessary? couldn't you just register this thing with `zcml`?